setup.py

```diff
@@ -23,6 +23,7 @@ def _setup_long_description() -> Tuple[str, str]:
         'loguru',
         'numpy',
         'openai',
+        'pydantic>=2.0',
```

Comment:
What do you think if we will route the correct call in the `Serializable` class depending on the pydantic version and remove the `>=2.0` from the setup.py file?

Reply:
I'd prefer not to unless we have a specific use case/reason or active conflict we know we'll hit to support Pydantic < 2.0. This is to keep our test matrix smaller